openssl/src/nid.rs

```diff
@@ -104,7 +104,7 @@ pub enum Nid {
     G,
     S,
     I,
-    UID,
+    uid, // uniqueIdentifier
```

Comment:
Actually, it seems like this should keep its name and the new variant should be called `UserId` or something like that to preserve back compat, yes?